### ISSUE TITLE
No Search-Guard Management Option within Kibana

### ISSUE THREAD

**Issue by tvollscw** [deviantony/docker-elk#305]
Having an issue where Basic Authentication is working with Search-Guard and Kibana, but none of the Search-Guard Kibana Management Menus are appearing. I ran `docker-compose exec -T kibana bin/kibana-plugin list` and did verify that Search Guard was already an installed plugin.

Is there something else that I'm missing?
I didn't see anything else in the ReadMe pertaining to activating this feature in particular..

**Comment by antoineco** [comment]
That's unfortunately expected in the free version 
https://github.com/deviantony/docker-elk/issues/242#issuecomment-363818240

**Comment by tvollscw** [comment]
It is not; unless the demo displays more content than usual.
I've been able to create my own Elastic Stack outside of docker built on top of Ubuntu 16.04 and have no issues with seeing the Search Guard option within it when using the [Search-Guard Demo Installation instructions](https://docs.search-guard.com/latest/demo-installer). 

From what I've read, [Search-Guard needs configured access to the REST API](https://github.com/floragunncom/search-guard-kibana-plugin/issues/46)?

Just wanted to check here first before I tried to do some nitty-gritty work.
